packages/ckeditor5-engine/src/view/matcher.js

```diff
@@ -394,6 +462,13 @@ function matchStyles( patterns, element ) {
  *			}
  *		};
  *
+ *		// Match view element which has many custom data attributes.
+ *		const pattern = {
+ *			attributes: [
+ *				{ key: /data-.+/, value: true }
+ *			]
+ *		}
+ *
```

Comment:
Docs should be also extended with other new options, like `attributes: true` etc

Reply:
Good catch! Thanks!